### ISSUE TITLE
"Execute all" button fails, gives error

### ISSUE THREAD

**Issue by vijay-prema** [TeaM-TL/FotoKilof#90]
**Describe the bug**
Execute all function does not work for a folder of png images

**To Reproduce**
1. Open a png image from a folder of many png images.
2. Click on top left and bottom right corner of image preview for Crop tool to select region.
3. Press Execute to crop single image, works fine.
4. Click Execute all button to run the same Crop on all png images in folder (with "Folder" and ".png" options selected)
5. Imagesin folder do not get cropped and it throws error:
```
Exception in Tkinter callback
Traceback (most recent call last):
  File "/home/linuxbrew/.linuxbrew/opt/python@3.8/lib/python3.8/tkinter/__init__.py", line 1892, in __call__
    return self.func(*args)
  File "/home/vijay/.platformio/penv/lib/python3.8/site-packages/src/__main__.py", line 369, in apply_all_button
    if result == "OK":
UnboundLocalError: local variable 'result' referenced before assignment
```

**Expected behavior**
I expected the Crop to be applied to all png images in the folder (batch apply).

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Ubuntu (Gnome) 20.04.2 LTS x86_64
 - Kernel: 5.8.0-53-generic
 - FotoKilof: 3.5.7 (installed with pip, Python 3.8.8)

**Additional context**

**Comment by TeaM-TL** [comment]
Hi
many thanks for this finding.
I will investigate soon. It is real problem.

**Comment by TeaM-TL** [comment]
fixed in 3.5.9, try it
